### PR TITLE
fix github dependabot security issues

### PR DESCRIPTION
**Pull request by zhmolly11**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@types/react-router-dom": "^5.3.3",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
+    "elliptic": "6.5.5",
```
Comment (johnnyluo) [inline frontend/package.json:23]: image
I think the version should be larger than 6.5.6 to get it fix

**Review by johnnyluo — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (zhmolly11) [reply frontend/package.json:23]: no, 6.5.6 is the latest version and it has the issue.

Comment (johnnyluo) [reply frontend/package.json:23]: so 6.5.5 is good?

Comment (zhmolly11) [reply frontend/package.json:23]: yes

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**